### PR TITLE
subgenerator info: fix displaying file for windows

### PR DESCRIPTION
**Pull request by pascalgrimaud**
- use shelljs instead of exec, when displaying file

Can you test it plz @deepu105 ?

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
Here an example:

##### **JHipster Version(s)**

```
jhipster@0.0.0 /home/pgrimaud/projects/test/20-default
└── generator-jhipster@3.5.0  -> /home/pgrimaud/projects/jhipster/generator-jhipster

```

##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**

```yaml
{
  "generator-jhipster": {
    "jhipsterVersion": "3.5.0",
    "baseName": "jhipster",
    "packageName": "com.mycompany.myapp",
    "packageFolder": "com/mycompany/myapp",
    "serverPort": "8080",
    "authenticationType": "session",
    "hibernateCache": "ehcache",
    "clusteredHttpSession": "no",
    "websocket": "no",
    "databaseType": "sql",
    "devDatabaseType": "h2Disk",
    "prodDatabaseType": "mysql",
    "searchEngine": "no",
    "buildTool": "maven",
    "enableSocialSignIn": false,
    "rememberMeKey": "7a7a7b6b111f68ec3ffe8ce6d08936b5fd5a70ac",
    "useSass": false,
    "applicationType": "monolith",
    "testFrameworks": [
      "gatling"
    ],
    "jhiPrefix": "jhi",
    "enableTranslation": true,
    "nativeLanguage": "en",
    "languages": [
      "en"
    ]
  }
}
```

##### **Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

IssueRepo.json

```yaml
{
  "relationships": [],
  "fields": [
    {
      "fieldName": "sourceType",
      "fieldType": "IssueSourceType",
      "fieldValues": "JIRA,GITHUB,BITBUCKET",
      "fieldValidateRules": [
        "required"
      ]
    },
    {
      "fieldName": "name",
      "fieldType": "String"
    },
    {
      "fieldName": "url",
      "fieldType": "String",
      "fieldValidateRules": [
        "required"
      ]
    }
  ],
  "changelogDate": "20160722181438",
  "entityTableName": "issue_repo",
  "dto": "no",
  "pagination": "no",
  "service": "no"
}
```

SuperToto.json

```yaml
{
  "relationships": [],
  "fields": [
    {
      "fieldName": "name",
      "fieldType": "String"
    },
    {
      "fieldName": "url",
      "fieldType": "String",
      "fieldValidateRules": [
        "required"
      ]
    }
  ],
  "changelogDate": "20160722155755",
  "entityTableName": "super_toto",
  "dto": "no",
  "pagination": "no",
  "service": "no"
}
```

##### **Browsers and Operating System**

openjdk version "1.8.0_91"
OpenJDK Runtime Environment (build 1.8.0_91-8u91-b14-0ubuntu4~16.04.1-b14)
OpenJDK 64-Bit Server VM (build 25.91-b14, mixed mode)

git version 2.7.4

node: v4.4.7

npm: 3.10.5

bower: 1.7.9

gulp:
[23:14:41] CLI version 1.2.1
[23:14:41] Local version 3.9.1

yeoman: 1.8.3

Docker version 1.11.1, build 5604cbe

docker-compose version 1.7.1, build 0a9ab35

**Comment by deepu105** [comment]
Ill check today

**Comment by deepu105** [comment]
its working

File: generators/info/index.js [hunk 1]
```diff
@@ -21,39 +21,32 @@ module.exports = ReportGenerator.extend({
...
-                if (!err) {
-                    this.log('```yaml');
-                    this.log(stdout);
-                    this.log('```\n');
-                }
-                done();
-            }.bind(this));
+            var result = shelljs.cat('.yo-rc.json');
+            this.log('##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**\n' +
+                '\n```yaml\n' + result + '\n```\n');
```
Comment (deepu105) [inline generators/info/index.js]: shouldnt this be ```json?

Comment (deepu105) [inline generators/info/index.js]: ok yaml seems to hv better highlighting, github should do better for json :P

**Pull request merged**